FrontEnd/src/components/ProfilePage/FormComponents/DeleteProfileComponent/DeleteProfileModal.js

```diff
@@ -12,7 +16,15 @@ const DeleteProfileModal = (props) => {
     const submitHandler = (event) => {
         event.preventDefault();
         props.onCancel();
-        // TODO logic of deleting
+        axios.delete(`${process.env.REACT_APP_BASE_API_URL}/api/profiles/${props.user.profile_id}`)
```

Comment:
I think we want to `await` for the delete completion and then logout the user otherwise show an error

Reply:
I added logic that we will be logged out only after the profile has been successfully deleted, in a new commit.

Reply:
Did you push the commit? `axios.delete` is an async operation lines below execute immediately w/o waiting for the delete completion. 
https://javascript.info/async-await